docs/en/deploy-loom-testnet.md

```diff
@@ -10,13 +10,13 @@ This doc is for developers, and it aims to show how they can join the Loom Testn
 
 ## Overview
 
-Deploying to Loom PlasmaChain is no different from deploying to the Ethereum main net, or to Rinkeby. If you know how to do one, you also know how to do the other.
+Deploying to Loom Basechain is no different from deploying to the Ethereum main net, or to Rinkeby. If you know how to do one, you also know how to do the other.
 
-We at Loom are using Truffle to build, test, and deploy our smart contracts. To make our life easier, we developed something called a provider that lets Truffle deploy to Loom PlasmaChain just like it deploys to Ethereum.
+We at Loom are using Truffle to build, test, and deploy our smart contracts. To make our life easier, we developed something called a provider that lets Truffle deploy to Loom Basechain just like it deploys to Ethereum.
```

Comment:
We at Loom are using --> Loom Network uses
deploy to Loom Basechain --> deploy to Basechain